src/PortingAssistant.Client.Analysis/Utils/CodeEntityModelModelToCodeEntities.cs

```diff
@@ -0,0 +1,308 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Codelyzer.Analysis;
+using Codelyzer.Analysis.Model;
+using NuGet.Versioning;
+using PortingAssistant.Client.Model;
+using TextSpan = PortingAssistant.Client.Model.TextSpan;
+
+namespace PortingAssistant.Client.Analysis.Utils
+{
+    public static class CodeEntityModelModelToCodeEntities
```

Comment:
Rename to CodeEntityModelToCodeEntities maybe? The double Model is a little strange but very minor thing and not necessary.

Reply:
Agreed. Too many models :D